layer.yaml

```diff
@@ -0,0 +1,4 @@
+repo: https://github.com/juju-solutions/layer-hacluster.git
+includes:
+  - 'layer:basic'
+  - 'interface:hacluster'
```

Comment:
I would also include `layer:kubernetes-common` given that you depend on it [here](https://github.com/juju-solutions/layer-hacluster/pull/1/files#diff-9a253747c39ed7c15f13a65546ad5ba3R4)

Reply:
That just got pulled due to the comment below and so this dependency is going away.

Reply:
scratch that, we left it.